templates/include/glean/event.1.schema.json

```diff
@@ -1,17 +1,16 @@
 {
-  "type": "array",
-  "items": [
-    {
-      "type": "integer",
-      "minimum": 0
+  "type": "object",
+  "properties": {
```

Comment:
Let's add `"additionalProperties": false` here so these objects can't contain anything extra.  If we add something in the future, we'll want to add that explicitly and I *think* having something additional should be treated as an error.

But I also defer to @fbertsch on that question.

Reply:
`"additionalProperties": false` added